src/lib/backups_service.ts

```diff
@@ -166,6 +164,13 @@ export class BackupsService extends ApplicationService {
     });
   }
 
+  private async _requestStoragePermissionsAndroid() {
```

Comment:
```suggestion
  private async requestStoragePermissionsAndroid() {
```

Underscore was before we were using TS I believe to signify private.

Reply:
Then we should remove underscores from other functions in this class, to keep consistency.